lib/config/defaults.js

```diff
@@ -988,26 +928,19 @@ const applyLoaderDefaults = (loader, { targetProperties }) => {
  * @param {WebpackNode} node options
  * @param {Object} options options
  * @param {TargetProperties | false} options.targetProperties target properties
- * @param {boolean} options.futureDefaults is future defaults enabled
  * @returns {void}
  */
-const applyNodeDefaults = (node, { futureDefaults, targetProperties }) => {
+const applyNodeDefaults = (node, { targetProperties }) => {
 	if (node === false) return;
 
-	F(node, "global", () => {
-		if (targetProperties && targetProperties.global) return false;
-		// TODO webpack 6 should always default to false
-		return futureDefaults ? "warn" : true;
-	});
+	D(node, "global", false);
 	F(node, "__filename", () => {
 		if (targetProperties && targetProperties.node) return "eval-only";
-		// TODO webpack 6 should always default to false
-		return futureDefaults ? "warn-mock" : "mock";
+		return false;
 	});
 	F(node, "__dirname", () => {
 		if (targetProperties && targetProperties.node) return "eval-only";
-		// TODO webpack 6 should always default to false
-		return futureDefaults ? "warn-mock" : "mock";
+		return false;
```

Comment:
I think will be good to add these changes in readme and how to return previously behavior